cmd/node/config.go

```diff
@@ -50,3 +53,13 @@ func (cfg *NodeConfig) allFieldsPresent() error {
 	}
 	return nil
 }
+
+func (config *NodeConfig) generateNodeID() (string, error) {
```

Comment:
@stepanvalyavskiy we definitely will need a binary representation of node id, so I'd return `[]byte` here, and convert to hex string when needed